src/Input/Input.js

```diff
@@ -561,13 +561,13 @@ Input.propTypes = {
    */
   onChange: PropTypes.func,
   /**
-   * TODO
```

Comment:
I don't see much value at exposing `onEmpty` and `onFilled`. I'm keeping them here for now in case someone is using it and because the tests are. Hopefully, we will remove it completely in the future.